Resources/config/hydra.xml

```diff
@@ -22,10 +28,12 @@
             <tag name="kernel.event_listener" event="kernel.response" method="onKernelResponse" />
         </service>
 
-        <service id="api.hydra.listener.request_exception" class="Dunglas\ApiBundle\Hydra\EventListener\RequestExceptionListener">
-            <argument type="service" id="api.hydra.normalizer.error" />
+        <service id="api.hydra.listener.exception" class="Symfony\Component\HttpKernel\EventListener\ExceptionListener">
+            <argument>api.hydra.controller.exception:showAction</argument>
```

Comment:
If think his idea was to normalize the exception, which makes sense. But if the exception do not come from the API system, it should redirect to the original `twig.exception_listener.controller`. See [this comment](https://github.com/dunglas/DunglasApiBundle/pull/299/files#r41495728).